src/loaders.ts

```diff
@@ -2,6 +2,15 @@ import { addressWithChecksum, fetchJSON } from "./utils.js";
 
 export interface ABILoader {
   loadABI(address: string): Promise<any[]>;
+  getContract(address: string): Promise<ContractData | null>
+}
+
+export interface ContractData {
+  abi: any[];
+  name: string | null;
```

Comment:
How do you feel about null here? Previously defaulted to an empty string.

Reply:
Fine with me